src/core/lib/iomgr/exec_ctx.h

```diff
@@ -28,10 +28,10 @@
 #include "src/core/lib/gpr/tls.h"
 #include "src/core/lib/iomgr/closure.h"
 
-typedef gpr_atm grpc_millis;
+typedef int64_t grpc_millis;
```

Comment:
Can this be uint64_t instead?

Reply:
Negative `grpc_millis` can indicate "past" time instead of future (I don't think we use currently negative `grpc_millis` anywhere though..but something useful to have)

Reply:
Yeah, as a general rule we should avoid unsigned unless it's mandated elsewhere (e.g., size_t). This is consistent with the [style guide](https://google.github.io/styleguide/cppguide.html#Integer_Types).